cfme/tests/configure/test_access_control.py

```diff
@@ -3024,4 +3051,35 @@ def test_ansible_playbook_stdout():
             9.
             10. able to see standard output
     """
-    pass
+    user, ansible_catalog = user_catalog
+    service_catalog = ServiceCatalogs(appliance, ansible_catalog, ansible_catalog_item.name)
+    ansible_service = MyService(appliance, ansible_catalog_item.name)
+
+    with user:
+        with appliance.context.use(ViaSSUI):
+            appliance.server.login(user)
+            view = ssui_nav(service_catalog, "Details")
+            view.add_to_shopping_cart.click()
+            service_catalog.order()
```

Comment:
You don't need to navigate to details page and click on shopping cart, you can just do `service_catalog.order()`.
```suggestion
            service_catalog.order()
```

Reply:
We have to explicitly add service in the catalog `service_catalog.order()` don't add